Sources/Tools/MaxExport/plExportErrorMsg.h

```diff
@@ -81,21 +66,6 @@ class plExportErrorMsg : public plErrorMsg {
 #else // PL_NULL_ERRMSG
 
 class plExportErrorMsg : public plErrorMsg {
-public:
-    plExportErrorMsg(const char* label, const char* msg) : plErrorMsg() { }
-    plExportErrorMsg(bool bogus = false) : plErrorMsg() { }
-    plExportErrorMsg(bool bogus, const char* label, const char* msg) 
-        : plErrorMsg() { }
-    plExportErrorMsg(bool bogus, const char* label, const char* format, const char* str) 
-        : plErrorMsg() { }
-    plExportErrorMsg(bool bogus, const char* label, const char* format, const char* str1, const char* str2) 
-        : plErrorMsg() { }
-    plExportErrorMsg(bool bogus, const char* label, const char* format, int n) 
-        : plErrorMsg() { }
-    plExportErrorMsg(bool bogus, const char* label, const char* format, int n, int m) 
-        : plErrorMsg() { }
-    plExportErrorMsg(bool bogus, const char* label, const char* format, float f) 
-        : plErrorMsg() { }
```

Comment:
These overrides called the empty base constructor (in order to suppress the messages) -- That should be carried over with the new constructor overloads.

Reply:
It looks like you fixed the wrong version -- this one's still empty, and the one that used to call the base constructors above now calls the empty constructors :(